idaes/commands/extensions.py

```diff
@@ -101,18 +117,28 @@ def get_extensions(
     library_only,
     no_download,
     show_current_version,
-    show_platforms):
+    show_platforms,
+    show_extras,
+    extras_only,
+    extra,
+    to):
 
     if show_platforms:
-        click.echo("\nBuild platforms for IDAES binary Extensions.  Most Linux")
-        click.echo("platforms are interchangeable.")
-        for key, mes in idaes.config.known_binary_platform.items():
+        click.echo("\nSupported platforms for IDAES binary extensions.")
+        for key, mes in sorted(idaes.config.known_binary_platform.items()):
             click.echo("    {}: {}".format(key, mes))
```

Comment:
very minor point, but f-strings are supported by all out Python versions now, so you could write: `click.echo(f"     {key}:{mes}")` instead